### DISCUSSION TITLE
Section combination & CPU boost

### DISCUSSION THREAD

**Discussion by MrGys** [seerge/g-helper#142] [Ideas]
1- I found myself clicking  "performance" and "gpu" section most of the time and they are sitting at the top whereas keyboard and animetrix I'm clicking less often.
Would be nice to combine "laptop keyboard"  and "anime matrix" into one section e.x (peripherals) as an extra pop up menu same as in "Fans + Power" and bring down the performance and gpu mode.
2- Also another idea would be to make CPU boost value saved uniquely for each Performance mode.

**Comment by seerge** [comment]
1. As for sections - I personally use animatrix dropdown a lot :)
2. As for boost - you should check https://github.com/seerge/g-helper/releases/tag/v0.40
